enterprise/reporting/event-stats/stats.go

```diff
@@ -0,0 +1,35 @@
+package event_stats
+
+import (
+	"strconv"
+
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	"github.com/rudderlabs/rudder-server/utils/types"
+)
+
+const (
+	EventsDeliveredMetricName = "events_delivered_total"
+	EventsAbortedMetricName   = "events_aborted_total"
+)
+
+func Report(workspaceId string, metric *types.PUReportedMetric) {
```

Comment:
@gane5hvarma please
1. Create a new Reporter implementation
2. Add `WorkspaceID` in `types.PUReportedMetric` and make sure it is being properly filled everywhere

Reply:
Sure @atzoum. I need some clarity on this, i will reach out to you on monday on this one